src/Service.Tests/SqlTests/RestApiTests/RestBodyNonStrictModeTests/RestBodyNonStrictModeTests.cs

```diff
@@ -237,17 +237,16 @@ public virtual async Task PatchOneWithExtraneousFieldsInRequestBody()
                 ""copies_sold"": 50,
                 ""last_sold_on_date"": ""2023-09-13 17:37:20""
             }";
-            expectedLocationHeader = $"id/3";
 
             await SetupAndRunRestApiTest(
-                    primaryKeyRoute: expectedLocationHeader,
+                    primaryKeyRoute: $"id/3",
                     queryString: null,
                     entityNameOrPath: _entityWithReadOnlyFields,
                     sqlQuery: GetQuery("PatchOneInsertWithComputedFieldInRequestBody"),
                     operationType: EntityActionOperation.UpsertIncremental,
                     requestBody: requestBody,
                     expectedStatusCode: HttpStatusCode.Created,
-                    expectedLocationHeader: expectedLocationHeader
+                    expectedLocationHeader: string.Empty
```

Comment:
Why are we not returning a location header for this request?

Reply:
I've  explained this in the [other comment](https://github.com/Azure/data-api-builder/pull/1731#discussion_r1332936049)